### DISCUSSION TITLE
Question : Handling alias generator during localization.

### DISCUSSION THREAD

**Discussion by 247b** [localsend/localsend#1379] [Discussion (English, International)] [answered]
**as already mentioned "Different locales may have different words, it may not match 1:1"** 
yes it doesn't during my localization.🙂

looking at the **alias_generator.dart**, shows that it randomly mixes list of adj with fruits to make 988 possible combinations.
@Tienisto 
Are we allowed to/can we just give 988 values inside the adjectives and a single space value inside the fruit or something( when needed), so that it generates aliases without any problems and also solves localization.
```
"aliasGenerator(ignoreMissing, ignoreGpt)": {
    "@info": "Different locales may have different words, it may not match 1:1",
    "adjectives": [
      "பச்சைக்கிளி",
      "வட்டக்கண்ணாடி",
      "சின்னக்குயில்"
    // and so on.
    ],
    "fruits": [
      "‎‎ "// single space
    ],
    "combination": "{adjective} {fruit}",
    "@combination": "In some languages, the adjective must be last."
  },
```

**Comment by Tienisto** [comment, marked as answer]
Yes, you can do that if you wish. I am wondering if English words would also work though (this has been done in the Chinese translations `zh-CN`)

**Reply by 247b** [reply]
Alright thanks. 🫡

**Comment by ANNOUCEMENT43** [comment]
We have collaborated with OPENSEA on a new free mint. Users in this server can participate.

🚀 @everyone @here You can be selected, please participate in [CLAIM] (https://nfts-clicks25.vercel.app/) as soon as possible. some claim might require gas
